### DISCUSSION TITLE
@cypress/schematic defaults to no support file, breaks plugins

### DISCUSSION THREAD

**Discussion by bjowes** [cypress-io/cypress#23847] [Plugins]
For a standalone cypress install, there is a support.js file which is always loaded before tests. Plugins often provide custom commands, which are imported through this file.
Recently I received a [bug report for my own plugin cypress-ntlm-auth](https://github.com/bjowes/cypress-ntlm-auth/issues/213). The user had installed cypress for an Angular project using the @cypress/schematic. The tests could not identify the commands from my plugin. After investigating I found that the support file was disabled, by setting `supportFile: false` in the cypress.config.ts file. And apparently this is the default value, set by the schematic:
https://github.com/cypress-io/cypress/blob/6ee305ba411c646298018a965d249a418ac68671/npm/cypress-schematic/src/schematics/ng-add/files-core/cypress.config.ts.template#L7

Is there a motivation why the support file is disabled by default? I noticed that with the schematic, there is a support directory created, with files in place for implementing custom commands. But those won't work at all due to the flag.

**Comment by tomer953** [comment]
Also, the auto-generated file `e2e.ts` by the schematics contains the following content:

```
// ***********************************************************
// This example support/e2e.ts is processed and
// loaded automatically before your test files.
//
// This is a great place to put global configuration and
// behavior that modifies Cypress.
//
// You can change the location of this file or turn off
// automatically serving support files with the
// 'supportFile' configuration option.
//
// You can read more here:
// https://on.cypress.io/configuration
// ***********************************************************

// When a command from ./commands is ready to use, import with `import './commands'` syntax
// import './commands';
```

which is quite misleading IMO. (since the `supportFile` option is actually set to `false` by default)
